01_Build_Process/02_Boot_Protocols.md

```diff
@@ -1,4 +1,4 @@
-# Boot Protocols
+[#](#.md) Boot Protocols
```

Comment:
I still see this odd syntax, `[#](#.md)`, does it have any particular purpose? Otherwise i think it can be removed.

Reply:
Sorry my editor auto adds it for some reason